other_recipes/jellof rice.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+   <h1>Jellof Rice</h1> 
+   <img src="hanxiao-lQN1CVzgQUw-unsplash.jpg"width="50%">
+   <h2>Description</h2>
+   <p>Jellof rice is a well known food eating by every body, it is very delicious when prepared very well</p>
+   <h2>Ingredients</h2>
+   <ul>
+   <li>5 medium sized Roma tomatoes, roughly chopped</li>
+   <li>1 red bell pepper, roughly chopped</li>
+   <li>1 medium sized onion, roughly chopped, set aside</li>
+   <li>2 scotch bonnet peppers (habanero peppers as they are sometimes called)</li>
+  <li>1/4 cup of groundnut oil</li>
+  <li>3 tbsp tomato paste</li>
+  <li>2 cups of parboiled rice</li>
+  <li>2 1/2 cups of chicken stock</li>
+  <li>1 tsp salt to taste</li>
+  <li>1/2 tsp curry powder</li>
+  <li>1/2 tsp thyme</li>
+  <li>1 tsp All purpose seasoning</li>
+  <li>1 Knorr stock cube</li>
+  <li>3 bay leaves</li>
+  <li>water, as needed</li>
+  <ul>
```

Comment:
Your `ul` closing tag should have a forward slash like this `</ul>`. Kindly fix this